pkg/lang/frontend/starlark/v1/config/config.go

```diff
@@ -225,3 +226,24 @@ func ruleFuncRepo(thread *starlark.Thread, _ *starlark.Builtin,
 	ir.Repo(url, description)
 	return starlark.None, nil
 }
+
+func ruleFuncOwner(thread *starlark.Thread, _ *starlark.Builtin,
+	args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var (
+		envdUid = starlark.MakeInt(-1)
+		envdGid = starlark.MakeInt(-1)
+	)
+
+	if err := starlark.UnpackArgs(ruleOwner, args, kwargs, "uid", &envdUid, "gid?", &envdGid); err != nil {
+		return nil, err
+	}
+	uid, _ := envdUid.Int64()
+	gid, _ := envdGid.Int64()
+	if uid < 0 || uid > 65535 || gid < 0 || gid > 65535 {
+		err := errors.New("get a wrong uid or gid")
+		return nil, err
+	}
```

Comment:
If `gid` is optional, seems it will fail because `gid=-1` by default.

Reply:
I think I should modify the optional properity.